### ISSUE TITLE
don't fail on non-existing (extended) parent templates

### ISSUE THREAD

**Issue by drzraf** [twigjs/twig.js#704]
The below test should no output anything. Instead it currently throws `Cannot extend an inline template.`

```js
        it('"extends" omits non-existent ascendants', function () {
            twig({
                id: 'parent.twig',
                data: '{% extends "non-existing.twig" %}'
            }).render().should.equal("");
        });
```

**Comment by justafish** [comment]
This should fix it #754

**Comment by willrowe** [comment]
It is not possible to extend a template that doesn't exist. This raises an error [in TwigPHP as well](https://twigfiddle.com/ven585).
